e2e-cypress/files/support/e2e.ts

```diff
@@ -15,3 +16,13 @@ afterEach(function() {
 
   consoleLogs.splice(0);
 })
+
+Cypress.Commands.add('addScreenshot', (title: string, screenshot: string) => {
```

Comment:
I'd rename anything "screenshot" here to "context path", e.g., `addScreenshot`->`addContextPath`.